framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -175,33 +174,39 @@ public static void main(String[] args) throws Exception {
   }
 
   public void generateReports(List<DrillTest> tests, int iteration) {
+
     try{
-      if(drillReportDirName == null){
-        drillReportDirName = CWD;
+      if(drillReportsDir == null){
+        drillReportsDir = CWD;
       }
-      File drillReportDir = new File(drillReportDirName);
+
+      File drillReportDir = new File(TestDriver.drillReportsDir);
+      FileSystem localFS = FileSystem.getLocal(conf);
+      FileSystem DFS = FileSystem.get(conf);
+
       if (!drillReportDir.exists()) {
         if (!drillReportDir.mkdir()) {
-          LOG.debug("Cannot create directory " + drillReportDirName
+          LOG.debug("Cannot create directory " + TestDriver.drillReportsDir
                   + ".  Using current working directory for drill output");
-          drillReportDirName = CWD;
+          TestDriver.drillReportsDir = CWD;
         }
       }
-      File reportFile = new File(drillReportDirName + "/" + version + "_" + commitId + "." + 
+
+      File reportFile = new File(TestDriver.drillReportsDir + "/apache-drill-" + version + "_" + commitId + "_" +
               "report_" + new Date().toString().replace(' ', '_').replace(':','_') + ".json");
+
       BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(reportFile));
       Document document;
       for (DrillTest test : tests) {
         document = Json.newDocument();
         document.set("_id", test.getTestId()+ "_" + new File(test.getInputFile()).getName() + "_" + test.getCloneId() + "_" + iteration);
-        document.set("testId", test.getTestId());
-        document.set("queryFilepath", test.getInputFile());
+        document.set("queryFilepath", test.getInputFile().substring(test.getInputFile().indexOf("resources/")+10));
```

Comment:
No - look carefully. What this is doing is to only consider the meaningful portion of the path - that's relative to the framework/resources directory. 

For example:
`/home/user1/drill-test-framework/framework/resources/Functional/suite/query.sql` is only saved as `Functional/suite/query.sql`. It makes it more readable.